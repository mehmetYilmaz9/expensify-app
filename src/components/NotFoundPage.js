import React from 'react';
import {Link} from 'react-router-dom';

const NotFoundPage = () => 
    //go back to page Home without refreshing the page
    (
        <div>
            <p>Love404!!!! </p>
            <Link to="/">GoHomeVieJS</Link>
        </div>
    );


export default NotFoundPage;