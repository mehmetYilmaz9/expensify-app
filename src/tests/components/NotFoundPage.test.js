import React from 'react';

//Shallow rendering is useful to constrain yourself to testing a component as a unit, 
//and to ensure that your tests aren't indirectly asserting on behavior of child components.
import { shallow } from 'enzyme';

import NotFoundPage from '../../components/NotFoundPage';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

test ('should render NotFoundPage with expenses', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});