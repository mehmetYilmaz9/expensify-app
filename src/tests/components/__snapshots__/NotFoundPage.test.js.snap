// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage with expenses 1`] = `
<div>
  <p>
    Love404!!!! 
  </p>
  <Link
    to="/"
  >
    GoHomeVieJS
  </Link>
</div>
`;
